components/Sections/Expertise.jsx

```diff
@@ -1,15 +1,15 @@
 import expertiseList from "../../utils/expertiseData";
```

Comment:
this ../../ it not necessary if you remove them there will be no change

Reply:
Really? 
I didn't know about that. let me try

Reply:
It works